testserver.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+import sys
+
+def noop_gettext(s):
+    return s
+
+gettext = noop_gettext
+
+HELPER_SETTINGS = dict(
+    LANGUAGES=(
+        ('en', u'English'),
+        ('de', u'Deutsch'),
+        ('it', u'Italiano'),
+        ('zh-cn', u'Chinese (Simplified)'),
+    ),
+    LANGUAGE_CODE='en',
+    PARLER_LANGUAGES={
+        1: (
+            {'code': 'en', 'fallbacks': ['de',]},
+            {'code': 'de', 'fallbacks': ['en',]},
+            {'code': 'it', 'fallbacks': ['en',]},
+            {'code': 'zh-cn', 'fallbacks': ['en',]},
+        ),
+        'default': {
+            'fallback': 'en',
+            'hide_untranslated': False,
+        },
+    },
+    PARLER_ENABLE_CACHING=False,
+    CMS_CACHE_DURATIONS={
+        'menus': 0,
+        'content': 0,
+        'permissions': 0,
+    },
+    # required for integration tests
+    LOGIN_URL='/admin/login/?user-login=test',
+    CMS_LANGUAGES={
+        1: [
+            {
+                'code': 'en',
+                'name': gettext('English'),
+                'fallbacks': ['de',],
+            },
+            {
+                'code': 'de',
+                'name': gettext('German'),
+                'fallbacks': ['en',],
+            },
+            {
+                'code': 'it',
+                'name': gettext('Italian'),
+                'fallbacks': ['en',],
+            },
+            {
+                'code': 'zh-cn',
+                'name': gettext('Chinese Simplified'),
+                'fallbacks': ['en',]
+            },
+        ],
+        'default': {
+            'fallbacks': ['en', 'de',],
+            'redirect_on_fallback': False,
+            'public': True,
+            'hide_untranslated': False,
+        }
+    },
+    INSTALLED_APPS=[
+        'reversion',
+        'djangocms_text_ckeditor',
+        'djangocms_grid',
+        'easy_thumbnails',
+        'filer',
+        'mptt',
+        'aldryn_bootstrap3',
+        'cms.test_utils.project.placeholderapp',
+    ],
+    MIDDLEWARE_CLASSES=[
+        'cms.middleware.utils.ApphookReloadMiddleware',
+    ],
+    TEMPLATE_DIRS=(
+        os.path.join(
+            os.path.dirname(__file__),
+            'cms', 'test_utils', 'project', 'templates', 'integration'),
```

Comment:
you're likely missing this path, so if tests would involve pages and not admin things would fail